woocommerce/woocommerce
# CYS: Remove not necessary placeholder on homepage sidebar

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





I noticed that the "classic" homepage sidebar displays an opt-in tracking message even when WooCommerce tracking is disabled. This appears to be a remnant of our previous iteration process. This PR removes the unnecessary tracking message.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you're using WordPress 6.5.
2. Visit `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`
3. Ensure that the tracking isn't enabled.
4. Head over to WooCommerce > Home > Customize your store.
5. Click "Choose Fonts".
6. Click "Design your Homepage"
7. Ensure that `Wants more pattern` copy isn't visible

![u3gk1Z.png](https://github.com/user-attachments/assets/eaf0b836-b1d9-4595-a0d6-4f6592490663)

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Remove not necessary placeholder on homepage sidebar.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
